index.html

```diff
@@ -43,5 +45,195 @@ <h1>Hey There. I'm Carlyle</h1>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card card6">
+          <div class="card-content">
+            <div class="cardtitle">
+              <p class="ct-text">Profesional Art Printing Data</p>
+            </div>
+            <div class="cardagraph">
+              <p class="cp-text">A daily selection of privately personalized reads; no accounts or sign-ups required. Due to my wider area, I decided to add more dummy text of my own making so as to appreciate the design more.</p>
+            </div>
+            <div class="cardtags">
+              <ul class="tags">
+                <li class="tag htag cardtagbg"><a href="#">html</a></li>
+                <li class="tag bootag cardtagbg"><a href="#">bootstrap</a></li>
+                <li class="tag rubytag cardtagbg"><a href="#">Ruby</a></li>
+              </ul>
+            </div>
+            <div class="fullwidth">
+              <button type="button" class="probtn1">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="info">
+    <div class="about">
+      <h2 class="am-title">About Me</h2>
+      <p class="am-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button type="button" class="am-btn">Get My Resume</button>
+      <span class="divider"></span>
+    </div>
+    <div class="expertise">
+        <div class="chialangs">
+          <div class="diamond"></div>
+          <div class="langnames">
+            <div class="languages"><p class="langtext">Languages</p></div>
+            <ul class="langlist">
+              <li class="lang">Javascript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">Html</li>
+              <li class="lang">Css</li>
+            </ul>
+          </div>
+        </div>
+        <div class="chiaframes">
+          <div class="square"></div>
+          <div class="framenames">
+            <div class="frameworks"><p class="frametext">Frameworks</p></div>
+            <ul class="framelist">
+              <li class="fram">Bootstrap</li>
+              <li class="fram">Ruby on Rails</li>
+              <li class="fram">RSPec</li>
+              <li class="fram">CapyBara</li>
+              <li class="fram">Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="chiaskills">
+          <div class="circle"></div>
+          <div class="skillnames">
+            <div class="skills"><p class="skilltext">Skills</p></div>
+            <ul class="skilllist">
+              <li class="ican">Codekit</li>
+              <li class="ican">Github</li>
```

Comment:
- [ ] [Required] Try to remove the space to the right of the page also. It will enhance the look and UX of your portfolio.
![image](https://user-images.githubusercontent.com/45668340/148662124-93b4d5ee-2e90-4271-9e64-89da1e88a511.png)